docker-compose.yml

```diff
@@ -3,12 +3,12 @@ version: '2'
 services:
 
   gateway-config:
-    image: nebraltd/hm-config:951749c
+    image: nebraltd/hm-config:8070d87
     depends_on:
       - dbus-session
       - diagnostics
     environment:
-      - FIRMWARE_VERSION=2021.11.11.0
+      - FIRMWARE_VERSION=2021.11.11.0-1
```

Comment:
I'd like to ensure that this non standard version number layout doesn't cause any issues with diagnostics and config

Don't see why it would, but just in case

Reply:
What's the way we've done it in the past? `2021.11.11.0.1`?

Reply:
Would have been 2021.11.11.1

I think your way is the best. Just want to make sure it doesn't cause any issues with anything. Can't see why it would in particular was just a thought

Reply:
I am happy to sign this off, the only issue I am seeing on the testnet is NebraLtd/hm-pktfwd#67 which should not be a blocker.
This helps a significant amount of units

Reply:
We treat it as a string, so this really shouldn't cause any issues and it has passed QA.

Reply:
@vpetersson we didn't QA it with this version number...

Reply:
@shawaj Should be covered in the QC playbook already (correct me if i'm wrong @KevinWassermann94).

Reply:
@KevinWassermann94 we can solve that by signing off a particular commit hash.